modules/core/include/opencv2/core/hal/hal.hpp

```diff
@@ -187,6 +187,25 @@ CV_EXPORTS void addWeighted32s( const int* src1, size_t step1, const int* src2,
 CV_EXPORTS void addWeighted32f( const float* src1, size_t step1, const float* src2, size_t step2, float* dst, size_t step, int width, int height, void* scalars );
 CV_EXPORTS void addWeighted64f( const double* src1, size_t step1, const double* src2, size_t step2, double* dst, size_t step, int width, int height, void* scalars );
 
+struct DftContext
+{
+    void * impl;
+    bool useReplacement;
```

Comment:
Yes, it is used in all cases (init/run/free functions, custom/original implementations): more details in [this block](https://github.com/mshabunin/opencv/blob/689b0311835d5678cc029d5d4233ae4b39215758/modules/core/src/dxt.cpp#L3222-L3340)